modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/CalciteBasicSecondaryIndexIntegrationTest.java

```diff
@@ -647,7 +650,7 @@ public void testOrderByNameCityDesc() {
     @Test
     public void testOrderByNoIndexedColumn() {
         assertQuery("SELECT * FROM Developer ORDER BY age DESC")
-            .matches(containsTableScan("PUBLIC", "DEVELOPER"))
```

Comment:
possibly we need to append additional check for IndexScan usage here and all appropriate tests? If it`s beyond the scope of this ticket may be we need additional one?

Reply:
We don't. Every test should have only those checks which verifies a state the current test is suppose to verify.